docs/learn/distributed-authority-quick-start.md

```diff
@@ -32,10 +34,24 @@ Before you begin, you need the following:
 During alpha and beta, you need to request access to the distributed authority service. To do so, provide your Unity contact with the ID of the Unity Cloud project you created.
```

Comment:
We can safely remove this note. The service is currently globally available with no restrictions.